### PR TITLE
Comms consoles now only work on the station zlevel.

### PR DESCRIPTION
**Pull request by Jc1469**
Not technically a fix as opposed to an outright change, I guess.  but w/e

#1678

:cl:
tweak: We're shit and so are our communications consoles. They now only work on the station.
/:cl:

### CODE REVIEW

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -50,9 +49,10 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 /obj/machinery/computer/communications/Topic(href, href_list)
 	if(..())
 		return
-	if (src.z > ZLEVEL_CENTCOM) //Can only use on centcom and SS13
-		usr << "<span class='boldannounce'>Unable to establish a connection</span>: \black You're too far away from the station!"
-		return
+	if(z_lock) //Can only be used on SS13 unless z_lock is changed..
+		if(z != 1)
```
Comment (X-TheDark) [inline code/game/machinery/computer/communications.dm]: `if(z != ZLEVEL_STATION)` is what you're looking for

**Review by X-TheDark — CHANGES_REQUESTED** [review]
Please use the defines whenever possible. Not a big deal though.

File: code/game/machinery/computer/communications.dm [hunk 2]
```diff
@@ -21,6 +19,7 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 	var/message_cooldown = 0
 	var/ai_message_cooldown = 0
 	var/tmp_alertlevel = 0
+	var/z_lock = 1//Var that admins can change to 0 if they need to use a comms console for some crap, I dunno.
```
Comment (Super3222) [inline code/game/machinery/computer/communications.dm]: if this is going to be Boolean use TRUE instead 1.

**Review by Super3222 — CHANGES_REQUESTED** [review]
bool

**Review by X-TheDark — APPROVED** [review]

**Review by Super3222 — APPROVED** [review]

**Comment by Super3222** [comment]
Looks good. I would've suggested letting emags break the z lock, but even that would bring up the same problems. In the future, we could make a centcomm version with a broken z lock and etc. Anyway it looks good.

@Cruix 

EDIT: No more memey commits or I'm closing your PR's.

**Comment by Jc1469** [comment]
aylmao recall seems pretty detailed to me.  aylmaos and recalling.  it's a good commit name, it gets to the point instantly.

**Comment by Super3222** [comment]
The first set of commit names aren't meant to be names. They are meant to get a clear point across to other coders in case they need to experiment, review, or debug earlier code.

**Comment by Super3222** [comment]
The first set of commit names aren't meant to be memes. They're meant to get a clear point across to other coders in case they need to experiment, review, or debug earlier code.

**Pull request merged**